python_scripts/trees_classification.py

```diff
@@ -169,14 +173,26 @@ def plot_decision_function(fitted_classifier, range_features, ax=None):
 # class predicted when the culmen depth is inferior to the threshold.
 
 # %%
-tree.predict([[0, 15]])
+sample_1 = pd.DataFrame(
+    {
+        "Culmen Length (mm)": [0],
+        "Culmen Depth (mm)": [15],
+    }
```

Comment:
Would it fit on the line? I have the impression that this is short enough.

```suggestion
    {"Culmen Length (mm)": [0], "Culmen Depth (mm)": [15]}
```